### PR TITLE
Adding MS Teams

### PR DESCRIPTION
**Pull request by davidaustinarcher**
Added payload for MS Teams as this gets used regularly.

### CODE REVIEW

**Comment by kaitlynmortimer** [comment]
@kristianamitchellcontrastsecurity Does this info overlap with your upcoming PR?

**Comment by kristianamitchellcontrastsecurity** [comment]
I have a current Teamserver PR that is waiting approval. This is set up so its ready for when that is approved and the integration goes live.

Kristiana Mitchell | Full Stack Engineer, Integrations
Contrast Security, Inc.<https://www.contrastsecurity.com/>
Welcome to the Era of Self-Protecting Software

**Comment by gmoore** [comment]
@davidaustinarcher We have a full-blown integration coming up with Microsoft Teams.

https://github.com/Contrast-Security-OSS/docs/pull/784

I believe that our work will supersede the webhook PR you submitted. Can you double-check to make sure that's the case? Thanks.

File: content/admin/org_settings/integrations/GenericWebhook.md [hunk 1]
```diff
@@ -41,7 +41,19 @@ If you were using the Generic Webhook to create a VictorOps integration, the sam
 	"state_message":"$Message" 
 }
 ```
- 
+
+Similarly you can set up an integration with Microsoft Teams using the following payload:
```
Comment (evamill) [inline content/admin/org_settings/integrations/GenericWebhook.md]: Simplify? Punctuation? Like:

You can integrate with Microsoft Teams in the same way. Use the following JSON payload:

**Comment by gmoore** [comment]
@davidaustinarcher Hold this and don't merge please until we know when we're releasing Teams.

**Comment by evamill** [comment]
Tagging this PR "on hold." @davidaustinarcher Please let us know when it's ready to merge. Copying @terra4383

**Comment by evamill** [comment]
@davidaustinarcher Is this work needed now that we have #824 PR. Tried asking @kristianamitchellcontrastsecurity but didn't get an answer. @gmoore seem to be saying your PR isn't needed now. Please confirm. Copying @gdvdsn-contrast

**Comment by gmoore** [comment]
@evamill Please close the request. https://github.com/Contrast-Security-OSS/docs/pull/824 is going out this week.

**Comment by davidaustinarcher** [comment]
No I think you can close it now we have a native integration.


On 13 Dec 2019, at 19:46, evamill <notifications@github.com> wrote:

﻿

@davidaustinarcher<https://github.com/davidaustinarcher> Is this work needed now that we have #824<https://github.com/Contrast-Security-OSS/docs/pull/824> PR. Tried asking @kristianamitchellcontrastsecurity<https://github.com/kristianamitchellcontrastsecurity> but didn't get an answer. @gmoore<https://github.com/gmoore> seem to be saying your PR isn't needed now. Please confirm. Copying @gdvdsn-contrast<https://github.com/gdvdsn-contrast>

**Pull request closed without merging**